src/config.rs

```diff
@@ -280,8 +280,8 @@ macro_rules! cf_config {
             pub compaction_style: DBCompactionStyle,
             pub disable_auto_compactions: bool,
             pub disable_write_stall: bool,
-            pub soft_pending_compaction_bytes_limit: ReadableSize,
-            pub hard_pending_compaction_bytes_limit: ReadableSize,
+            pub soft_pending_compaction_bytes_limit: Option<ReadableSize>,
```

Comment:
what's the output of `Option` after serialization

Reply:
`None` will not be serialized, `Some` is serialized in the same way as normal fields.